rotkehlchen/db/dbhandler.py

```diff
@@ -583,6 +574,20 @@ def set_settings(self, write_cursor: 'DBCursor', settings: ModifiableDBSettings)
         )
         CachedSettings().update_entries(settings)
 
+    def get_cache(self, cursor: 'DBCursor') -> DBCache:
+        cursor.execute('SELECT name, value FROM key_value_cache;')
+        cache_dict = {}
+        for q in cursor:
+            cache_dict[q[0]] = q[1]
+        return DBCache.deserialize(cache_dict)
+
+    def set_cache(self, write_cursor: 'DBCursor', cache: DBCache) -> None:
+        cache_dict = cache.serialize()
+        write_cursor.executemany(
+            'INSERT OR REPLACE INTO key_value_cache(name, value) VALUES(?, ?)',
+            list(cache_dict.items()),
+        )
```

Comment:
don't create variables that are used only once. Also add a small docstring

```suggestion
        write_cursor.executemany(
            'INSERT OR REPLACE INTO key_value_cache(name, value) VALUES(?, ?)',
            list(cache.serialize().items()),
        )
```